Pod/Classes/SEGMixpanelIntegration.m

```diff
@@ -139,6 +140,26 @@ - (void)screen:(SEGScreenPayload *)payload
     }
 }
 
+
+- (void) group:(SEGGroupPayload *)payload {
+    
+    NSString *groupID = payload.groupId;
+    NSDictionary *traits = [payload traits];
+    NSString *groupName = traits[@"name"];
+    
+    if(groupName == nil || groupName.length == 0) {
+        groupName = @"[Segment] Group";
+    }
```

Comment:
The JIRA ticket referenced looking at our a.js implementation for some guidance. Mixpanel required some group identifier and then a groupId to be used. Not all customers will want to use `traits.name` as their ideentifier. We have implemented a setting for the Mixpanel, `groupIdentifierTraits`, to help solve this issue. The description of the setting is "What trait Segment should use as your Mixpanel "group key" in group calls. If, for example, you set this to be `company`, then "company" will be sent as `group_key` and the value of `traits["company"]` will be sent as the `group_id`."

Here is a link to the a.js implementation for reference: https://github.com/segmentio/analytics.js-integrations/blob/1d99533c84ffa845eb2188d783d8775e7f8a8b09/integrations/mixpanel/lib/index.js#L325-L344. Can we update the code to leverage this setting instead of relying on `groupName`?

Reply:
Sure, will do that.